### ISSUE TITLE
looking for  obfio/twitter-tid-generator

### ISSUE THREAD

**Issue by funky-mango** [obfio/twitter-ui_mentrics#4]
your website points to the project but it's not there

https://github.com/obfio/twitter-tid-generator?ref=antibot.blog

did it get removed?

**Comment by NCLnclNCL** [comment]
https://github.com/yeyuchen198/twitter-tid-generator
![image](https://github.com/user-attachments/assets/79f37921-e880-47a8-b762-dfc12394982b)
 how to get frames ?

**Comment by orthofashion** [comment]
@NCLnclNCL, hi bro, do you know how to get key and frames?

**Comment by vlinh128** [comment]
yeah, same question, really looking forward to the answer @NCLnclNCL @obfio @orthofashion @yeyuchen198

**Comment by NCLnclNCL** [comment]
> @NCLnclNCL, hi bro, do you know how to get key and frames?



> yeah, same question, really looking forward to the answer @NCLnclNCL @obfio @orthofashion @yeyuchen198

https://github.com/iSarabjitDhiman/XClientTransaction
